foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -1057,12 +1063,11 @@ inline void Server<ServerConfiguration>::sendMessage(ConnHandle clientHandle, Ch
 
   const auto bufferSizeinBytes = con->get_buffered_amount();
   if (bufferSizeinBytes + payloadSize >= _options.sendBufferLimitBytes) {
-    FOXGLOVE_DEBOUNCE(
-      [this]() {
-        _server.get_elog().write(
-          WARNING, "Connection send buffer limit reached, messages will be dropped...");
-      },
-      2500);
+    const auto logFn = [this, clientHandle]() {
+      sendStatusAndLogMsg(clientHandle, StatusLevel::Warning,
+                          "Connection send buffer limit reached, messages will be dropped...");
```

Comment:
As a user I'd find it more helpful if this said something like "Server send buffer full. Dropping message on topic /foo/bar".

Reply:
I agree that this would be more helpful, but due to log debounce, you wouldn't see it for every topic for which messages are dropped